addons/purchase_requisition/tests/test_purchase_requisition.py

```diff
@@ -403,3 +403,34 @@ def test_12_alternative_po_line_different_currency(self):
         self.assertEqual(len(best_price_ids), 2)
         self.assertEqual(len(best_date_ids), 2)
         self.assertEqual(len(best_price_unit_ids), 2)
+
+    def test_13_recompute_date_planned(self):
+        """Create a blanket order + an RFQ for it"""
+        requisition_type = self.env["purchase.requisition.type"].create(
+            {"name": "Blanket test", "quantity_copy": "none"}
+        )
+
+        bo_form = Form(self.env["purchase.requisition"])
+        bo_form.vendor_id = self.res_partner_1
+        bo_form.type_id = requisition_type
+        with bo_form.line_ids.new() as line:
+            line.product_id = self.product_09
+            line.product_qty = 5.0
+            line.price_unit = 21
+        bo = bo_form.save()
+        bo.action_in_progress()
+
+        # lazy reproduction of clicking on "New Quotation" act_window button
+        po_form = Form(
+            self.env["purchase.order"].with_context(
+                {"default_requisition_id": bo.id, "default_user_id": False}
+            )
+        )
+        po_form.order_line.remove(0)
+        with po_form.order_line.new() as line:
+            line.product_id = self.product_09
+            line.product_qty = 5.0
+            line.price_unit = 21
+        po = po_form.save()
+        po.button_confirm()
+        self.assertEqual(po.state, "purchase")
```

Comment:
Perhaps you can cut the part about removing and re-adding the product's pol and paste it into the end of the existing test `test_03_blanket_order_rfq` (and move when the button_confirm occurs in the existing test) since the rest of this test repeats what is already in the existing test.

Reply:
Done :+1: